src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/todo';
+import { User } from './types/user';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todosList: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosList);
+  const [newTodo, setNewTodo] = useState('');
```

Comment:
```suggestion
  const [todoTitle, setTodoTitle] = useState('');
```

Reply:
Why todo - is an object of Todo interface and newTodo - is a string?😅 
Use more descriptive names